site/public/js/directory.js

```diff
@@ -5,7 +5,7 @@
 function newDownloadForm() {
     $('.popup-form').css('display', 'none');
     const form = $('#download-form');
-    form.css('display', 'block');
+    showPopup('#download-form');
```

Comment:
For all the JS files referencing this method, just add the method to the global comment at the top of the file like /* global ... showPopup */ and these errors will go